code/modules/paperwork/paperbin.dm

```diff
@@ -28,20 +28,13 @@
 	. = ..()
 
 /obj/item/paper_bin/attack_hand(mob/user)
-	if(!Adjacent(user))
```

Comment:
This check is necessary since, iirc, dragging and dropping it onto yourself from any distance will call `attack_hand()`.